apps/server/src/modules/school/dto/migration.dto.ts

```diff
@@ -0,0 +1,16 @@
+export class MigrationDto {
```

Comment:
why not calling it OAuthMigrationDto?

Reply:
Good point, but it's not necessary linked to oauth.